packages/openid4vc/tests/openid4vc-presentation-during-issuance.e2e.test.ts

```diff
@@ -161,7 +161,10 @@ describe('OpenId4Vc Presentation During Issuance', () => {
     issuer.agent.x509.addTrustedCertificate(issuer.certificate.toString('base64'))
 
     // We let AFJ create the router, so we have a fresh one each time
-    expressApp.use('/oid4vci', issuer.agent.modules.openId4VcIssuer.config.router)
+    expressApp.use(
+      '/oid4vci',
+      (issuer.agent.dependencyManager.registeredModules['openId4VcIssuer'] as OpenId4VcIssuerModule).contextRouter
```

Comment:
This should be easier, as users of the frameworks have to do this registration step as well. It should not rely on dependency injection or type casting. this is also why the router was available on the module config (where I think it should stay)